components/compliance-service/ingest/pipeline/publisher/node_manager_publisher.go

```diff
@@ -90,6 +91,10 @@ func gatherInfoForNode(in message.Compliance) (*manager.NodeMetadata, error) {
 		tags = append(tags, &common.Kv{Key: "environment", Value: in.Report.GetEnvironment()})
 	}
 
+	mgrType := in.Report.GetAutomateManagerType()
+	if in.Report.GetSourceFqdn() != "" {
```

Comment:
get the type on the report, but if the sourcefqdn is not empty string then we know it's managed by chef